seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -355,6 +355,22 @@ def test_validate_anvil_vcf(self, mock_file_iter, mock_file_exist, mock_utils_lo
         self.assertEqual(response.url,
                          '/login/google-oauth2?next=/api/create_project_from_workspace/my-seqr-billing/anvil-no-project-workspace1/validate_vcf')
 
+    @mock.patch('seqr.views.apis.anvil_workspace_api.get_gs_file_list')
+    def test_get_anvil_vcf_list(self, mock_get_file_list, mock_utils_logger):
+        # Requesting to load data from a workspace without an existing project
+        url = reverse(get_anvil_vcf_list, args=[TEST_WORKSPACE_NAMESPACE, TEST_WORKSPACE_NAME1])
+        self.check_manager_login(url, login_redirect_url='/login/google-oauth2')
+        mock_utils_logger.warning.assert_called_with('User does not have sufficient permissions for workspace {}/{}'
+                                                     .format(TEST_WORKSPACE_NAMESPACE, TEST_WORKSPACE_NAME1),
+                                                     self.collaborator_user)
+
+        # Test valid operation
+        mock_get_file_list.return_value = ['gs://test_bucket/test.vcf', 'gs://test_bucket/data/test.vcf.gz']
```

Comment:
you should add some non-VCF files to the mock list to test that they are properly filtered out